clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -2979,14 +2993,21 @@ drmemtrace_get_timestamp_from_offline_trace(const void *trace, size_t trace_size
 
 void
 raw2trace_t::add_to_statistic(raw2trace_thread_data_t *tdata, raw2trace_statistic_t stat,
-                              int value)
+                              uint64 value)
 {
     switch (stat) {
     case RAW2TRACE_STAT_COUNT_ELIDED: tdata->count_elided += value; break;
     case RAW2TRACE_STAT_DUPLICATE_SYSCALL: tdata->count_duplicate_syscall += value; break;
     case RAW2TRACE_STAT_FALSE_SYSCALL: tdata->count_false_syscall += value; break;
     case RAW2TRACE_STAT_RSEQ_ABORT: tdata->count_rseq_abort += value; break;
     case RAW2TRACE_STAT_RSEQ_SIDE_EXIT: tdata->count_rseq_side_exit += value; break;
+    case RAW2TRACE_STAT_EARLIEST_TRACE_TIMESTAMP:
+        tdata->earliest_trace_timestamp =
+            std::min(tdata->earliest_trace_timestamp, value);
```

Comment:
I found it confusing to see the call above which says "add": can we rename this function to "merge_statistic" or something along those lines -- I think that will make it easier to understand when seeing just the calls at line 823, 823.

Reply:
Done. Used "accumulate" instead of "merge"

Reply:
Hmm, "accumulate" sounds like "add" to me: keeping just the max or min doesn't fit with "accumulate" IMHO.